UKHomeOffice/nmsw-ui
# Test people count

## AC

Test to show count of people on FAL5 and 6 after uploaded
Test to view list of passengers on passenger's status check page


## To test
```
npm run cypress:open:dev -- --env MAIL_API_KEY=<API_KEY>
```
and select 
1. check-your-answers-page.feature
2. passenger-status-check.feature
or run in cl
```
npm run cypress:run:dev -- --spec cypress/e2e/features/check-your-answers-page.feature
```
```
npm run cypress:run:dev -- --spec cypress/e2e/features/passenger-status-check.feature

```
